hack/build.sh

```diff
@@ -72,8 +72,8 @@ if [[ "${GOOS}" = "windows" ]]; then
   OUTPUT="${OUTPUT}.exe"
 fi
 
-go build \
+GO111MODULES=on go build \
```

Comment:
Since we are using golang:1.14 as the base container, do we still need to turn on go module functionality explicitly?

Reply:
It looks like it still defaults to "auto" which gives some problems when building in the gopath with something that does need modules, so the current way we build Astrolabe wound up not working if GO111MODULE=on is not set.